src/components/auth/torus/AuthTorus.js

```diff
@@ -64,7 +64,7 @@ const AuthTorus = ({ screenProps, navigation, styles, store }) => {
       log.debug('reinitializing wallet and storage with new user')
       goodWallet.init()
       await goodWallet.ready
-      await userStorage.init()
+      userStorage.init()
```

Comment:
```suggestion
      userStorage.init()
      await userStorage.ready()
```

probably we should await once userStorage will re-initialize again ?

Reply:
i have await userStorage.ready just a couple of lines later